.github/workflows/go.yml

```diff
@@ -67,6 +66,42 @@ jobs:
         run:  go test -race -v ./...
         working-directory: go/src/github.com/openconfig/ygot
 
+      - name: Check that Go integration test files compile
+        working-directory: go/src/github.com/openconfig/ygot/ygen/testdata
+        run: |
+          function test-go-build() {
+            cwd=$(pwd)
+            tmpdir=$(mktemp -d -p $cwd)
+            name=$(basename $1)
+            cp "$1" "$tmpdir"/"${name}.go"
+            cd $tmpdir
+            goimports -w *.go
+            go build
+            cd $cwd
+            rm -r $tmpdir
+          }
+
+          expectFail=(
```

Comment:
Why do we expect that these fail? If we're outputting anything shouldn't it compile?

Reply:
Thanks for mentioning this, I've added comments detailing the reasons in the CI script.

Reply:
In both the cases below, should we just fix these? It seems like:

* the unsupported symbol shouldn't be output in the code
* the default unions case should return an error (or produce code that compiles - with a caveat that says the default isn't supported)

Reply:
Fixed.